src/routes/createevent.js

```diff
@@ -0,0 +1,20 @@
+const createEvent = require('../handlers/createevent')
+const validateEvent = require('../models/validateEvent.js')
+
+module.exports = (req, res) => {
+  validateEvent(req)
+    .then(() => createEvent(req.body))
+    .then(event => {
+      if (!event) {
+        throw new Error('There was a problem adding the event to the database')
+      } else {
+        res.redirect(`/event/${event.title}`)
+      }
+    })
+    .catch((err) => {
```

Comment:
could we put this ```.catch``` attached to ```validateEvent```? At the moment it is at the end, meaning it will also render any error message coming directly from the DB. Might be better to have a generic message for rendering in a last ```.catch```.